README.md

```diff
@@ -4,7 +4,7 @@
 **Tags:** pantheon, cdn, cache  
 **Requires at least:** 4.7  
 **Tested up to:** 6.1
-**Stable tag:** 1.2.0
+**Stable tag:** 1.2.1
```

Comment:
Is latest release not 1.2.2? Something is off here, might need a reconcile of master/develop/branch?

Reply:
`develop` does not get the version bump automagically, since releases are merged into `master`. there might be some automation/process stuff to do there, but in the meantime the versions need to come from `master`. Apparently there had been a release since the time that I created the PR.